registry/storage/driver/rados/rados.go

```diff
@@ -471,6 +471,7 @@ func (d *driver) Delete(ctx context.Context, objectPath string) error {
 		totalSize, err := d.getXattrTotalSize(ctx, oid)
 
 		if err != nil {
+			err = d.deleteOid(objectPath)
```

Comment:
This needs to be more clear for the intent. Do you intend to hide the error and return nil if the delete is successful or do you want to try to delete object path and hide the other error if that fails, returning success otherwise?